src/iterator.rs

```diff
@@ -585,33 +589,38 @@ mod tokio_support {
         ///
         /// # Examples
         ///
-        /// ```rust
+        /// ```rust,norun
```

Comment:
Why is this norun? Won't it just not compile at all if we don't have the dependency anyway?

Reply:
I changed this to `rust,ignore` locally but forgot to push it.